### PR TITLE
[PREVIEW] ROC-3935 Example dates

### PR DESCRIPTION
**Pull request by andrewwa-kainos**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-3935

### Change description
Add 'addDays(n)' filter to dynamically mutate Moments in templates
Use new filter for outstanding hard-coded example dates
(Except birthdate.)

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/features/offer/views/your-offer.njk [hunk 1]
```diff
@@ -10,16 +10,19 @@
       <form novalidate method="post">
         {{ csrfProtection(csrf) }}
 
-        <p>{{ t('Make it clear what you will do and by when. For example: "I will pay you £240 on 29 April 2018 by bank transfer".') }}</p>
+        <p>{{ t('Make it clear what you will do and by when. For example: "I will pay you £240 on {{ nextMonth }} by bank transfer".',
+            {nextMonth: 'now' | addDays(30) | inputDate}) }}</p>
 
-        <p>{{ t('If you’re also asking {{ claimantName }} to do something, say clearly what it is and by when it should be done. For example: "You will fix the roof by 28 April 2018."', { claimantName: claim.claimData.claimant.name }) }}</p>
+        <p>{{ t('If you’re also asking {{ claimantName }} to do something, say clearly what it is and by when it should be done. For example: "You will fix the roof by {{ twoWeeksLater }}."',
+            { claimantName: claim.claimData.claimant.name, twoWeeksLater: 'now' | addDays(44) | inputDate }) }}</p>
```
Comment (geek-soft) [inline src/main/features/offer/views/your-offer.njk]: Adding 44 days does not make it two weeks later

**Comment by geek-soft** [comment]
I am very happy with filter based solution

File: src/main/features/ccj/views/repayment-plan.njk [hunk 2]
```diff
@@ -26,7 +26,8 @@
         <div class="form-group">
           <fieldset>
             <legend class="visually-hidden">{{ t('Date for first payment') }}</legend>
-            {{ dateInput('firstPaymentDate', form, hint = 'For example, 12 03 2017',
+            {{ dateInput('firstPaymentDate', form,
+            hint = t('For example, {{ nextMonth }}', {nextMonth : 'now' | addDays(30) | inputDate}),
```
Comment (timja) [inline src/main/features/ccj/views/repayment-plan.njk]: what's `inputDate` do vs `date`?

File: src/main/features/offer/views/your-offer.njk (hunk 1 above)
Comment (andrewwa-kainos) [reply src/main/features/offer/views/your-offer.njk]: I meant two weeks after the 30 days from the previous calculation

File: src/main/features/ccj/views/repayment-plan.njk (hunk 2 above)
Comment (andrewwa-kainos) [reply src/main/features/ccj/views/repayment-plan.njk]: `inputDate` formats it as "dd MM yyyy" (31 7 2018) whereas `date` formats it as "dd MMMM yyyy" (31 July 2018)

File: src/main/features/offer/views/your-offer.njk (hunk 1 above)
Comment (timja) [reply src/main/features/offer/views/your-offer.njk]: why not just 14 days?

**Comment by andrewwa-kainos** [comment]
Awaiting PO approval before merging

**Comment by andrewwa-kainos** [comment]
Demoed to Marc James with approval

**Pull request merged**